pkg/apis/numaflow/v1alpha1/get_spec_req.go

```diff
@@ -113,9 +113,11 @@ type GetMonoVertexPodSpecReq struct {
 }
 
 type GetServingPipelineResourceReq struct {
-	ISBSvcConfig     BufferServiceConfig         `protobuf:"bytes,1,opt,name=isbSvcConfig"`
-	Image            string                      `protobuf:"bytes,2,opt,name=image"`
-	PullPolicy       corev1.PullPolicy           `protobuf:"bytes,3,opt,name=pullPolicy,casttype=k8s.io/api/core/v1.PullPolicy"`
-	Env              []corev1.EnvVar             `protobuf:"bytes,4,rep,name=env"`
-	DefaultResources corev1.ResourceRequirements `protobuf:"bytes,5,opt,name=defaultResources"`
+	ISBSvcConfig                  BufferServiceConfig         `protobuf:"bytes,1,opt,name=isbSvcConfig"`
+	Image                         string                      `protobuf:"bytes,2,opt,name=image"`
+	PullPolicy                    corev1.PullPolicy           `protobuf:"bytes,3,opt,name=pullPolicy,casttype=k8s.io/api/core/v1.PullPolicy"`
+	Env                           []corev1.EnvVar             `protobuf:"bytes,4,rep,name=env"`
+	DefaultResources              corev1.ResourceRequirements `protobuf:"bytes,5,opt,name=defaultResources"`
+	Replicas                      *int32                      `protobuf:"varint,6,opt,name=replicas"`
```

Comment:
nit: Can get these 2 directly from the `ServingSpec`, do not need to be set in the Req.

Reply:
Done